src/screens/SendConfirmScreen.js

```diff
@@ -75,23 +75,35 @@ class SendConfirmScreen extends React.Component {
    * @param {String} pin User PIN already validated
    */
   executeSend = (pin) => {
-    const ret = this.props.sendTx(this.props.wallet, this.amount, this.address, this.token, pin);
-    if (ret.success) {
-      // show loading modal
-      this.setState({
-        modal:
-          // eslint-disable-next-line react/jsx-indent
-          <SendTransactionFeedbackModal
-            text={t`Your transfer is being processed`}
-            sendTransaction={ret.sendTransaction}
-            successText={<TextFmt>{t`Your transfer of **${this.amountAndToken}** has been confirmed`}</TextFmt>}
-            onDismissSuccess={this.exitScreen}
-            onDismissError={() => this.setState({ modal: null })}
-          />,
-      });
+    const outputs = [{ address: this.address, value: this.amount, token: this.token.uid }];
+    let sendTransaction;
+    // TODO substitute for feature flag when rollout code is implemented
+    if (false) {
```

Comment:
This is the second time I see this code. I feel we should move it to an util method (or pass it as `this.props.sendTransaction` from mapStateToProps).

Reply:
Created a redux variable that is being used now as feature flag. https://github.com/HathorNetwork/hathor-wallet-mobile/pull/111/commits/6f6c05de5d4714040ee97ffdd2dcddf37e9ac619

Reply:
Great! Thanks!